src/Operators/operator2stencil.jl

```diff
@@ -33,7 +75,7 @@ get_boundary(op::Operator2Stencil, bw::RightBoundaryWindow{name}) where {name} =
     get_boundary(op.op, bw)
 
 function return_eltype(op::Operator2Stencil, args...)
-    lbw, ubw = stencil_interior_width(op.op, args...)[end]
+    lbw, ubw = stencil_interior_width(op.op, args...)[1]
```

Comment:
Is this one of the bug fixes?

Reply:
This is actually the extension of `Operator2Stencil` to advection and flux-correction operators. I initially linearized every operator with respect to its last argument (because that is what determines the operator's name; i.e., C2F, C2C, etc.), but these operators need to be linearized with respect to velocity, which is their first argument. So, I am now linearizing every operator with respect to its first argument.